.drone/drone.yml

```diff
@@ -190,6 +190,38 @@ depends_on:
 - docker-amd64
 - docker-arm64
 
+---
+kind: pipeline
+name: cd-to-dev-env
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: image-tag
+  image: alpine/git
+  commands:
+  - apk --update --no-cache add bash
+  - git fetch origin --tags
+  - echo $(./tools/image-tag) > .tags
+
+- image: us.gcr.io/kubernetes-dev/drone/plugins/updater
```

Comment:
I noticed this step is missing `name`, but I can't find if this is required for a step. Maybe just add it to be sure?